CONTRIBUTORS.md

```diff
@@ -1,41 +0,0 @@
-# Contributing to dl4j-examples
-
-Thanks for your interest in Deeplearning4j's examples. Our goal is to bring fast, open-source deep learning to all JVM-based communities.
-
```

Comment:
I think we should probably keep files like this around.

Reply:
Done